contracts/mock/MockKlerosLiquid.sol

```diff
@@ -0,0 +1,385 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+/* solhint-disable */
+// A standalone contract that mocks below
+// https://github.com/kleros/kleros/blob/master/contracts/kleros/KlerosLiquid.sol
+// Names starting with __ do not exist in KlerosLiquid and added for convenience here.
+// Events are omitted for brevity.
+// Set the metadata to start with subcourt 1 with 3 minimum number of jurors.
+// All staking/voting functionality is stripped, use __setCurrentRuling() instead.
+// Use __setSubcourtTimesPerPeriod() if you need to speed up the flow.
```

Comment:
I see the name of the implementing function is `changeSubcourtTimesPerPeriod`, I guess it should be `__setSubcourtTimesPerPeriod`?

Reply:
I did initially use `__setSubcourtTimesPerPeriod()`, but then noticed that there is already a `changeSubcourtTimesPerPeriod()` in KlerosLiquid so I overrode it. `__setSubcourtTimesPerPeriod()` may make more sense though.